### DISCUSSION TITLE
teachers mouse pointer not shown on clients

### DISCUSSION THREAD

**Discussion by nektarios1201** [epoptes/epoptes#81] [Q&A]
teachers mouse pointer not shown on clients, so clients cannot see where the teachers ponts.

thanks

Nektarios

**Comment by alkisg** [comment]
Hi Nektarios, I've fixed this 2 months ago in https://github.com/Epoptes/epoptes/commit/193a105e44f412f26a30e1589e196dfcc05443e5.

Which epoptes version are you using? `dpkg -l epoptes`
Which vnc viewers do you have installed? `dpkg -l '*vnc*' | grep ^ii`

**Comment by jcrisolf** [comment]
I have the same problem.
Latest version of epoptes for ubuntu 18.04: epoptes 1.0.1-2+t201 all  
VNC version: 
libvncclient1:amd64 0.9.11+dfsg-1ubuntu1 amd64        API to write one's own VNC server - client library
libvncserver1:amd64 0.9.11+dfsg-1ubuntu1 amd64        API to write one's own VNC server
tigervnc-viewer     1.7.0+dfsg-8ubuntu2  amd64        Virtual network computing client for X
x11vnc              0.9.13-6             amd64        VNC server to allow remote access to an existing X session
x11vnc-data         0.9.13-6             all          data files for x11vnc

Thanks
Joaquim

**Comment by alkisg** [comment]
Hi, you do have the latest Ubuntu version but you don't have the latest Epoptes version.
To get the fix, you need to use the Epoptes stable PPA which currently has Epoptes =  1.0.1-2+t201811221803~ubuntu18.04.1.

Documentation: http://www.epoptes.org/documentation/ppa

**Comment by jcrisolf** [comment]
I do have the stable repository and the latest version (1.0.1-2+t201811221803~ubuntu18.04.1) but the mouse pointer does'nt show on client compueters. Maybe it is because I use xfce4 desktop.

**Comment by alkisg** [comment]
Ah sorry, I stopped reading at the point where the version was cut off in your initial comment.

So, the problem is that you're using tigervnc viewer instead of xvnc4viewer. I don't see an option to show a local cursor in the [tigervnc viewer manpage](https://tigervnc.org/doc/vncviewer.html), so maybe you can switch to xvnc4viewer that does support "-UseLocalCursor=0".

Or you could file a bug report against tigervnc, to support showing a local cursor.

**Comment by nektarios1201** [comment]
Hi Alkis
Sorry, for the delay.
I installed the new version and it works.
The only thing that it could be better is the speed. It takes a couple of seconds for the pointer to appear on users screen if the distance of the move is long.
But that could be beacuse of my network speed?
Anyway thanks a lot

**Comment by jcrisolf** [comment]
Thanks a lot. I removed tigervnc-viewer and installed xvnc4viewer and now it works.
Joaquim
